vms/platformvm/service.go

```diff
@@ -588,11 +588,11 @@ type GetStakingAssetIDResponse struct {
 func (service *Service) GetStakingAssetID(_ *http.Request, args *GetStakingAssetIDArgs, response *GetStakingAssetIDResponse) error {
 	service.vm.SnowmanVM.Ctx.Log.Info("Platform: GetStakingAssetID called")
 
-	if args.SubnetID.IsZero() {
+	if args.SubnetID == ids.Empty {
```

Comment:
This can be removed.

Reply:
If we remove this line then we're assuming that the primary network ID will remain the empty ID, which I think is an OK assumption. I'll remove these